test/features/sdk_flow.feature

```diff
@@ -2,126 +2,160 @@
 Feature: SDK File Upload Tests
 
   Scenario Outline: I should be able to upload a passport and an image of a face correctly.
-    Given I verify with passport
+    Given I verify with passport with <locale>
     When I try to upload passport <type>
-    Then page_title () should contain "Selfie"
+    Then page_title should include "capture.face.upload_title"
     When I try to upload one_face
     Then I should reach the complete step
 
     Examples:
-      | type |
-      |      |
-      | pdf  |
+      | type | locale |
+      |      |        |
+      | pdf  |        |
```

Comment:
we can remove this line from all other examples too. It's enough to have two combinations of type and locale:
`['', '']` and `['pdf, 'es']`